apps/barista-design-system/src/pages/.search-page/search-page.module.ts

```diff
@@ -17,15 +17,15 @@
 import { CommonModule } from '@angular/common';
```

Comment:
search page is deprecated. You could delete it.

Reply:
Here: #1285